internal/backend/local/test.go

```diff
@@ -1004,7 +1004,7 @@ func (runner *TestFileRunner) cleanup(file *moduletest.File) {
 // more variables than are required by the config. FilterVariablesToConfig
 // should be called before trying to use these variables within a Terraform
 // plan, apply, or destroy operation.
-func (runner *TestFileRunner) GetVariables(config *configs.Config, run *moduletest.Run, references []*addrs.Reference) (terraform.InputValues, tfdiags.Diagnostics) {
+func (runner *TestFileRunner) GetVariables(config *configs.Config, run *moduletest.Run, references []*addrs.Reference, action string) (terraform.InputValues, tfdiags.Diagnostics) {
```

Comment:
```suggestion
func (runner *TestFileRunner) GetVariables(config *configs.Config, run *moduletest.Run, references []*addrs.Reference, includeWarnings bool) (terraform.InputValues, tfdiags.Diagnostics) {
```

I think it's better in this case to be clear about what the variable is actually controlling. It's just doing one thing at the moment and we can make the behaviour clear in the function signature with this.

Reply:
Thank you @liamcervante! Yes `includeWarnings` is much better than my string!